testing/scenario_app/ios/Scenarios/Scenarios.xcodeproj/project.pbxproj

```diff
@@ -451,27 +450,6 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
-/* Begin PBXShellScriptBuildPhase section */
-		0D8470A8241035AD0030B565 /* Compile iOS JIT */ = {
```

Comment:
it didn't make sense to me that we would auto build the dart code (which isn't even visible in the xcode project) when running xcode but not build the engine